manifests/0000_90_cluster-etcd-operator_02-dashboards.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    release.openshift.io/delete: "true"
+  labels:
+    console.openshift.io/dashboard: "true"
+  name: remove-etcd-dashboard
```

Comment:
I'm not entirely sure what that does, @Elbehery. This will create a new configmap with the name "remove-etcd-dashboard" that will then be removed by the CVO.

I think we need to patch the existing configmap `grafana-dashboard-etcd` with the annotation `release.openshift.io/delete: "true"`.

Reply:
thanks @tjungblu  for your prompt review 

So iiuc from my chat with @wking , if we add the annotation to the existing CM, all etcd-dashboards will be removed 

Currently we have two dashboard for etcd. I think adding a new CM with the annotation will remove one of them, please correct me if  i a wrong